### DISCUSSION TITLE
Backup of databases (not volume)

### DISCUSSION THREAD

**Discussion by mariusa** [caprover/caprover#1701] [Q&A] [answered]
Hi,

Trying to setup DB backups (not volume backups) for Postgres, similar to https://devcenter.heroku.com/articles/heroku-postgres-backups#scheduled-backups

I've read
https://github.com/caprover/caprover/issues/158
https://github.com/caprover/caprover/issues/410

https://caprover.com/docs/backup-and-restore.html is about entire disk volume, not applicable.

What's the current most simple solution to have Postgres just dump db backups in a folder & upload to S3 in the one-click app? eg have a crontab to run pg_dump / s3_cmd
All DB servers also need DB backups.

Thanks

**Comment by githubsaturn** [comment, marked as answer]
This is outside of CapRover scope. You can use any 3rd party postgres backup solution that is compatible with Docker. 

This one is apparently a good one: https://github.com/tiredofit/docker-db-backup
